angularSTEP/src/app/branching-recipe-form/branching-recipe-form.component.html

```diff
@@ -0,0 +1,124 @@
+<mat-horizontal-stepper linear>
+  <mat-step [stepControl]="basicsFormGroup">
+    <form [formGroup]="basicsFormGroup" (ngSubmit)="onSubmit()" class="form-formatting">
+      <ng-template matStepLabel>Recipe Basics</ng-template>
+        <mat-form-field>
+          <mat-label>Recipe Name</mat-label>
+          <input matInput placeholder="Ex. Mom's tamales" formControlName="name" autocomplete="off" required>
```

Comment:
I am thinking that this page is almost the same as the adding a recipe form. Thus, I will check over it but in less detail if it the same, since we have already looked over that fairly extensively. If not, please let me know, and I will go into a lot more depth when looking at this form.

Reply:
Yes most of it is the same as the original upload form

Reply:
Wouldn't this feature be way easier to implement and maintain if the existing recipe creation form was augmented to support pre-filling an existing recipe instead of always starting blank?

Reply:
I think originally when I was making the form I didn't want to accidentally remove something that was necessary for the functionality of the upload form as well as being able to compare the two side by side in different tabs when I was building up the branching form, but you make a good point.